src/step-function.ts

```diff
@@ -1272,6 +1273,49 @@ class BaseStandardStepFunction<
     };
   }
 
+  readonly apiGWVtl: ApiGatewayVtlIntegration = {
+    renderRequest: (call, context) => {
+      const args = retrieveMachineArgs(call);
+
+      return `{\n"stateMachineArn":"${
+        this.resource.stateMachineArn
+      }",\n${Object.entries(args)
+        .filter(
+          (arg): arg is [typeof arg[0], Exclude<typeof arg[1], undefined>] =>
+            arg[1] !== undefined
+        )
+        .map(([argName, argVal]) => {
+          if (argName === "input") {
+            // stringify the JSON input
+            return `"${argName}":"$util.escapeJavaScript(${context.exprToJson(
+              argVal
+            )})"`;
+          } else {
+            return `"${argName}":${context.exprToJson(argVal)}`;
+          }
+        })
+        .join(",")}\n}`;
+    },
+
+    createIntegration: (options) => {
+      const credentialsRole = options.credentialsRole;
+
+      this.resource.grantRead(credentialsRole);
+      this.resource.grantStartExecution(credentialsRole);
+
+      return new aws_apigateway.AwsIntegration({
+        service: "states",
+        action: "StartExecution",
+        integrationHttpMethod: "POST",
+        options: {
+          ...options,
+          credentialsRole,
+          passthroughBehavior: aws_apigateway.PassthroughBehavior.NEVER,
+        },
+      });
+    },
+  };
```

Comment:
Should we share this across express and standard?

Reply:
uhhgg, I was trying to be lazy and got caught.